intercom_flutter/android/build.gradle

```diff
@@ -41,6 +41,6 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    implementation 'io.intercom.android:intercom-sdk:15.1.4'
+    implementation 'io.intercom.android:intercom-sdk:15.1.6'
```

Comment:
Looks like the latest available version is `15.1.5`

Reply:
Hello @Zazo032, version `15.1.6` just went out : https://github.com/intercom/intercom-android/releases